tests/test_hf_utils.py

```diff
@@ -22,3 +24,27 @@ def test_load_tokenizer_in_memory_fs():
         )
     tokenizer = load_tokenizer("memory://foo/")
     assert len(tokenizer) == 5027
+
+
+@skip_if_hf_model_not_accessible("meta-llama/Llama-2-7b-hf")
```

Comment:
Should we add some tests with more funky Unicode characters and tokens that don't align on character boundaries?